### PR TITLE
[ADD] new module that shows the clasic 5 star rating on products

### PR DESCRIPTION
**Pull request by oscarolar**
Here is the [RUNBOT](http://runbot.vauxoo.com/runbot/team/yoytec-12#8_0-yoytec-website-rate-product)

### CODE REVIEW

File: website_rate_product/controllers/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from . import main
```
Comment (nhomar) [inline website_rate_product/controllers/__init__.py:1]: Falato

# -_\- coding: utf-8 -_-

File: website_rate_product/__init__.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+from . import models
+from . import controllers
```
Comment (nhomar) [inline website_rate_product/__init__.py:3]: Falta # -_\- coding: utf-8 -_-

**Comment by oscarolar** [comment]
Are you serious??? [LINK](https://github.com/Vauxoo/addons-vauxoo/blob/8.0/website_variants_extra/models/__init__.py)

**Comment by oscarolar** [comment]
If runbot is green here we are ready to merge.

**Pull request merged**